Geometry/VeryForwardGeometry/python/geometryRPFromDB_cfi.py

```diff
@@ -1,12 +1,19 @@
 import FWCore.ParameterSet.Config as cms
 
 XMLIdealGeometryESSource_CTPPS = cms.ESProducer("XMLIdealGeometryESProducer",
-                                                rootDDName = cms.string('cms:CMSE'),
-                                                label = cms.string('CTPPS'),
-                                                appendToDataLabel = cms.string('XMLIdealGeometryESSource_CTPPS')
-                                                )
+    rootDDName = cms.string('cms:CMSE'),
+    label = cms.string('CTPPS'),
```

Comment:
@forthommel - it looks like we did not decouple this as we've discussed. I'm not sure this will continue working with DD4hep in production since we lock the main geometry after it has been parsed.

Reply:
@ianna sorry for my ignorance. Is there any chance you still have a reference to the discussion thread so that I can see what could enter the scope of this PR (and what can possibly be introduced in a follow-up PR)?

Reply:
I am not sure what you mean by "decouple". I am not an expert for the DD4Hep option, but the result of the migration made by @ghugo83 seems to work. Do you expect any difference between the "private" tests and the application "in production"?

Reply:
@jan-kaspar - Detector description **by design is a source** to build desired geometries, it **must not** be a geometry itself. Neither `DD` nor `DD4hep` should allow usage of `XMLIdealGeometryESSource` as a RECO geometry. Besides, there is no guarantee it would work in a multi-threaded mode. It is `Xerces` implementation dependant, but neither `DocumentBuilderFactory` nor `DocumentBuilder` are guaranteed to be thread safe. It may work in a "private" test because you run in a single thread. It will most certainly fail "in production".

Reply:
Thanks for the details!

Our plan is that in the next few days @wpcarvalho pushes a PR making PPS reco geometry being loaded from DB directly. The use of ``XMLIdealGeometryESSource`` (or so) will be then isolated to the preparation of the DB payloads.